src/server/scripts/EasternKingdoms/zone_undercity.cpp

```diff
@@ -126,10 +116,23 @@ class npc_lady_sylvanas_windrunner : public CreatureScript
         uint32 BlackArrowTimer;
         uint32 ShotTimer;
         uint32 MultiShotTimer;
+        bool flag;
+        Player * _player;
```

Comment:
what happens if the player logs out after _player is set and before _player is checked ?